examples/example_plugin/example_plugin/views.py

```diff
@@ -77,6 +78,18 @@ class ExampleModelUIViewSet(views.NautobotUIViewSet):
     serializer_class = serializers.ExampleModelSerializer
     table_class = tables.ExampleModelTable
 
+    @action(detail=False, methods=["get"], url_path="custom_action_get_all_example_model_names")
+    def custom_action_get_all_example_model_names(self, request):
```

Comment:
Usually the action corresponds to the URL endpoint, so maybe something like `def all_names(...)` might be more typical?